### DISCUSSION TITLE
Hold note end in some skins in lazer for osu!mania look worse in quality than they should

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#25358] [Ideas] [answered]
### Type

Cosmetic

### Bug description

Stable:
![image](https://github.com/ppy/osu/assets/86934170/fa6cc330-e2f2-4b9b-a0bf-80da3219cd2d)
Lazer:
![image](https://github.com/ppy/osu/assets/86934170/d994fc7c-5e8f-4225-87c2-7844fc166c8e)

### Version

2023.1026

### Assets

[Hanabi ([Crz]Noire).zip](https://github.com/ppy/osu/files/13255680/Hanabi.Crz.Noire.zip)

**Comment by peppy** [comment, marked as answer]
There's no easy fix for this. What we have is best effort. The skins that do this are using 40,000 pixel textures which break GPUs.

You're welcome to attempt to implement a better solution, but a better option would be to wait for a skin api that doesn't use these hacks.
